bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/ExpandItem.java

```diff
@@ -310,7 +309,12 @@ public int getHeaderHeight () {
 }
 
 int getHeaderHeightInPixels () {
-	return Math.max (parent.getBandHeight (), imageHeight);
+	int headerHeightInPixels = parent.getBandHeight ();
+	int imageHeightInPixels = DPIUtil.scaleUp(imageHeight, getZoom());
+	if(imageHeightInPixels > headerHeightInPixels) {
+		headerHeightInPixels = imageHeightInPixels + headerHeightInPixels;
```

Comment:
Is this really intended? It looks wrong to me that in case the image is higher than the band of the expand bar, the header his is _the sum_ of these two. Shouldn't it just be the image height then?

I guess that's also the reason why there is some unexpected margin at the bottom and top of the image within the header when the image is larger than the header (text) itself:
![image](https://github.com/user-attachments/assets/da185aa4-f29f-4353-9905-a6a24f43735c)

When "fixing" this, it looks like this:
![image](https://github.com/user-attachments/assets/da8bcdec-8ca2-48ef-ab55-96d9cf704b88)

I do not want to say that this is better / intended, but just adding the `headerHeightInPixels` for a margin seems wrong to me. Wouldn't it be better to define some explicit margin that is added to the image height to ensure that the image does not start/end at the bounds of the header?

Reply:
You should test to see what also looks good in particular in border cases. Keep in mind that you should not apply the margin once the image height exceeds the header height but that you should ensure that the margin is always present, even if the image fits into the ordinary header height. Otherwise, an image with the height of the header will have no margin and an image with height+1 of the header will have margin of 10. So I would test what happens with according values for the margin in these border cases.

Reply:
I have decided on 8 to be the best margin. Also I have handled the case to have the margin of at least 8 when the image height is equal or closer to the header height.